tests/unit/test_req_install.py

```diff
@@ -42,3 +45,49 @@ def test_forward_slash_results_in_a_link(self, tmpdir):
         )
 
         assert requirement.link is not None
+
+    def test_install_req_from_string_without_comes_from(self):
```

Comment:
I don't think these tests belong in `TestInstallRequirementBuildDirectory`? Perhaps they should be at the top level as `install_req_from_req_string()` is a top-level function.

Reply:
Done.